crates/metal-mir/src/stmt/functiondef.rs

```diff
@@ -1,9 +1,13 @@
 use super::Statement;
 use crate::{parcel, types};
 
+/// Represents a definition for a function
 #[derive(Debug, Clone)]
 pub struct FunctionDefinition {
```

Comment:
Shouldn't this have a visibility field?

Reply:
Function signature includes a visibility field.